### ISSUE TITLE
request not match when using in abbreviated manifest

### ISSUE THREAD

**Issue by joebnb** [verdaccio/verdaccio#3357]
https://github.com/verdaccio/verdaccio/blob/7c12550a1409626b832730bbf5976464fe0b7d52/src/api/endpoint/api/package.ts#L11

![image](https://user-images.githubusercontent.com/1646883/188439939-5a1d4d65-2191-44f3-b520-256acb3de44d.png)

**Comment by juanpicado** [comment]
thanks @joebnb I'll check later.

ref: https://github.com/npm/feedback/discussions/749
ref: https://github.com/verdaccio/verdaccio/issues/2792

**Comment by juanpicado** [comment]
@joebnb please try with `verdaccio@5.15.2` thanks for reporting and early testing.

**Comment by joebnb** [comment]
this feature are trying to solve heaving manifest, so could verdaccio export a option to save and response mainfest without  spacing, i think it would be default,we look this fewer than computer request,npmjs did.

![image](https://user-images.githubusercontent.com/1646883/188774244-165b9b00-9526-4a6a-9946-4a3ad48a0272.png)
![image](https://user-images.githubusercontent.com/1646883/188774790-5872fa3f-edfe-4c85-b9f2-5a0f61bbec6f.png)

**Comment by juanpicado** [comment]
good point you are right, I think just applying this without arguments would be enough, I think should by default in both, I think this has been there for a while for unknown reasons.

**Comment by juanpicado** [comment]
Could you apply this patch in your local server https://github.com/verdaccio/verdaccio/pull/3364 and see if improve the situation in your environment?

**Comment by joebnb** [comment]
![image](https://user-images.githubusercontent.com/1646883/188803687-1b319219-056c-406f-94db-ef4f44e8be97.png)

it's workwell


btw manifest are specified with document may use this to pre-define a schema for better perform was another choice，but there are some works to do，this just a advice 
https://github.com/lucagez/slow-json-stringify/blob/HEAD/benchmark.md
this is a small test i wrote mouth ago,install benchmark and node ./this.js
```js
'use strict';

const benchmark = require('benchmark');
const suite = new benchmark.Suite();

const schema = {
    title: 'Example Schema',
    type: 'object',
    properties: {
        firstName: {
            type: 'string',
        },
        lastName: {
            type: 'string',
        },
        age: {
            description: 'Age in years',
            type: 'integer',
            minimum: 0,
        },
    },
    required: ['firstName', 'lastName'],
};

const obj = {
    firstName: 'test',
    lastName: 'fast',
    age: 32,
};

const stringify = (function build(schema) {
    let code = `
    'use strict'
    let json = '{'
  `;

    Object.keys(schema.properties).forEach((key, i, a) => {
        const type = schema.properties[key].type;

        code += `
      json += '"${key}":'
    `;

        switch (type) {
            case 'string':
                code += `
          json += '"' + obj.${key} + '"'
        `;
                break;
            case 'integer':
                code += `
          json += '' + obj.${key}
        `;
                break;
            default:
                throw new Error(`${type} unsupported`);
        }

        if (i < a.length - 1) {
            code += "json += ','";
        }
    });

    code += `
    json += '}'
    return json
  `;

    return new Function('obj', code);
})(schema);

suite.add('JSON.stringify', () => {
    JSON.stringify(obj);
});

suite.add('fast-json-stringify', () => {
    stringify(obj);
});

suite.on('complete', print);

suite.run();

function print() {
    for (var i = 0; i < this.length; i++) {
        console.log(this[i].toString());
    }

    console.log('Fastest is', this.filter('fastest').map('name')[0]);
}

```


in my issue there are alternate bug in caused function memo,please consider give a investigation.

**Comment by juanpicado** [comment]
I've shipped on v5.15.3 🚀 this one is on you 🥇 .

Regarding the suggestion, I'm agree, I've been also experimenting with https://github.com/davidmarkclements/fast-safe-stringify but I haven't research enough, let me checkout your suggestion and during this weeks I'll experiment a bit and I will ping you when I have something to show :) thanks so much.
